week-6/Homework/mandatory/2-quotegenerator/index.html

```diff
@@ -11,7 +11,25 @@
     />
     <link rel="stylesheet" href="style.css" />
   </head>
-  <body>
+  <body id=backgroundColorBody>
+    <div id=quoteDiv>
+      <p id=quoteParagraph>
+        <!-- "Whether you think you can or you think you can’t, you’re right.", -->
+      </p>
+      <p id=quoteAuthorParagraph>
+         <!-- "Henry Ford", -->
+      </p>
+      <button id=newQuoteButton>NewQuote</Button>
```

Comment:
```suggestion
      <button id=newQuoteButton>NewQuote</Button>
```
The button element is nested inside the `quoteDiv` and as a result is changing its position when the quote text changes. Think about how you could rearrange the html slightly so the button position is fixed on the page.